src/components/CampaignURLBuilder/Web/index.tsx

```diff
@@ -316,7 +313,6 @@ export const CampaignUrlBuilder: React.FC<{ version: GAVersion }> = ({
           </TableRow>
         </TableBody>
       </Table>
-
       <Typography variant="h2">Related Resources</Typography>
       <Typography variant="body1" component="ul">
         <li>
```

Comment:
Can you fix the links. Some are duplicates here.

Also, no related resource links for Android, iOS?

Reply:
Removed the duplicate link. 

I was going to wait on related resources for once I had a better sense of where these are linked to/from, etc.

Reply:
Update the links to remove the visit_id query parameter. There are 2 links in constants that include that param and shouldn't.